extras/ams_wrapper/src/config.py

```diff
@@ -20,62 +20,7 @@
 from src.api.models.classification_attributes_model import ClassificationAttributes
 from src.api.ovms_connector import OvmsConnector
 
-# Default version for a model is the latest one
-DEFAULT_VERSION = 0 
-
-AVAILABLE_MODELS = [
-    {"name": "vehicleDetection", 
-    "class": DetectionModel, 
-    "ovms_mapping": {
-        "model_name": "vehicle_detection_adas",
-        "model_version": DEFAULT_VERSION,
-        "input_name": "data",
-        },
-    "config_path": "/opt/ams_models/vehicle_detection_adas_model.json",
-    },
-    {"name": "vehicleClassification", 
-    "class": ClassificationAttributes, 
-    "ovms_mapping": {
-        "model_name": "vehicle_attributes",
-        "model_version": DEFAULT_VERSION,
-        "input_name": "input",
-        },
-    "config_path": "/opt/ams_models/vehicle_attributes_model.json"
-    },
-    {"name": "emotionsRecognition", 
-    "class": ClassificationAttributes, 
-    "ovms_mapping": {
-        "model_name": "emotions_recognition",
-        "model_version": DEFAULT_VERSION,
-        "input_name": "data",
-        },
-    "config_path": "/opt/ams_models/emotions_recognition_model.json"
-    },
-    {"name": "ageGenderRecognition", 
-    "class": ClassificationAttributes, 
-    "ovms_mapping": {
-        "model_name": "age_gender_recognition",
-        "model_version": DEFAULT_VERSION,
-        "input_name": "data",
-        },
-    "config_path": "/opt/ams_models/age_gender_recognition_model.json"
-    },
-    {"name": "personVehicleBikeDetection", 
-    "class": DetectionModel, 
-    "ovms_mapping": {
-        "model_name": "person_vehicle_bike_detection",
-        "model_version": DEFAULT_VERSION,
-        "input_name": "data",
-        },
-    "config_path": "/opt/ams_models/person_vehicle_bike_detection.json"
-    },
-    {"name": "faceDetection", 
-    "class": DetectionModel, 
-    "ovms_mapping": {
-        "model_name": "face_detection_adas",
-        "model_version": DEFAULT_VERSION,
-        "input_name": "data",
-        },
-    "config_path": "/opt/ams_models/face_detection_adas.json"
-    },
-]
+TYPE_CLASS_MAPPING = {
```

Comment:
Maybe we can move this definition elsewhere to remove the config.py file ? wrapper.py ?

Reply:
How about just renaming it? I think it would be good to have all AMS related constants in one place.

Reply:
Ok